install/kubernetes/Makefile

```diff
@@ -6,22 +6,13 @@ include ../../Makefile.defs
 # Workaround until we branch for v1.9
 VERSION := 1.8.90
```

Comment:
Since you are fixing this file, can you remove these 2 lines and rebase with master?